src/classes/OPP_OpportunityNaming_BATCH.cls

```diff
@@ -59,27 +76,45 @@ public class OPP_OpportunityNaming_BATCH implements Database.Batchable<sObject>,
     * @description Batch process execute method. Names and updates all opportunities in the current batch.
     */
     public void execute(Database.BatchableContext BC, List<sObject> scope) {
+        List<Opportunity> oppsToProcess = scope;
```

Comment:
An alternative to this is to change the method signature to:
`public void execute(Database.BatchableContext BC, List<Opportunity> oppsToProcess)`

Reply:
I didn't realize you could do that, I thought it would violate the signature for the `Database.Batchable<sObject>` interface. Thanks!